packages/python-packages/apiview-copilot/src/_apiview_reviewer.py

```diff
@@ -90,3 +119,179 @@ def _retrieve_static_guidelines(self, language, include_general_guidelines: bool
                 items = json.loads(f.read())
                 language_guidelines.extend(items)
         return general_guidelines + language_guidelines
+    
+    def _get_filter_expression(self) -> str:
+        """
+        Returns the filter expression for the given language.
+        """
+        if self.language == "all":
+            return "lang eq '' or lang eq null"
+        else:
+            return f"lang eq '{self.language}' or lang eq '' or lang eq null"
+
+    def _search_guidelines(self, query: str) -> List[object]:
+        credential = DefaultAzureCredential()
```

Comment:
can we use the same credential defined at the top of the file?

Reply:
Done